api/server/server.go

```diff
@@ -1313,8 +1330,9 @@ func createRouter(eng *engine.Engine, logging, enableCors bool, dockerVersion st
 			"/exec/{name:.*}/resize":        postContainerExecResize,
 		},
 		"DELETE": {
-			"/containers/{name:.*}": deleteContainers,
-			"/images/{name:.*}":     deleteImages,
+			"/containers/{name:.*}/logs": deleteContainersLogs,
```

Comment:
This is the original implementation, and seems appropriate since we are deleting logs, but will update to whatever y'all like.